README.md

```diff
@@ -30,9 +30,26 @@ The Migration module provides methods to manage database migrations in a structu
 
    `pip install -r requirements.txt`
 
+## Flask initialization
+
+In your Flask application, import the migration_blueprint from fhir_migrations.commands and register it with your Flask app. This provides routes for running migration commands:
+
+<pre>
+from flask import Flask
+from fhir_migrations.commands import migration_blueprint
+
+app = Flask(__name__)
+
+# Register the migration blueprint with your Flask app
+app.register_blueprint(migration_blueprint)
+</pre>
+
 ## Usage
 
-To initialize the Migration service and use a specific directory for migration scripts, you need to provide the path to the directory when creating an instance of the Migration class. By default, the service looks for migration scripts in the versions directory located in the same directory as the script.
+To initialize the Migration service and use a specific directory for migration scripts, you need to provide the path to the directory when creating an instance of the Migration class. By default, the service looks for scripts in examples directory, and can be altered via specifying MIGRATION_SCRIPTS_DIR enviroment var.  
+- For writing new migrations: the random UUID for the migration's revision id is generated automatically and does not need to be specified by user. The upgrade method is run on `upgrade` flask command, downgrade method is run on `downgrade` flask command. The method declaration for both is provided, but method functionality needs to be filled by the user.
```

Comment:
unless i'm overlooking it, you don't mention `how` to generate the migration script here?

Reply:
something like "see the `migration` command, below